nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/PutSQS.java

```diff
@@ -57,6 +59,14 @@
         + "the Message Attribute and value will become the value of the Message Attribute", expressionLanguageScope = ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
 public class PutSQS extends AbstractSQSProcessor {
 
+    public static final PropertyDescriptor CHARSET = new PropertyDescriptor.Builder()
```

Comment:
PutSQS.CHARSET appears similar to GetSQS.CHARSET.  Can this property descriptor be moved to AbstractSQSProcessor and shared?

Reply:
I thought about it during the implementation, but it will have an effect on the other SQS components. Lets put this into `AbstractSQSProcessor` to avoid duplications.